src/main/java/org/jenkinsci/plugins/badge/StatusImage.java

```diff
@@ -94,11 +96,11 @@
      */
     @Override
     public void generateResponse(StaplerRequest req, StaplerResponse rsp, Object node) throws IOException, ServletException {
-        String v = req.getHeader("If-None-Match");
+        /*String v = req.getHeader("If-None-Match");
```

Comment:
We may want to be careful. There was caching built into this repo to prevent a call to Jenkins every time.